### ISSUE TITLE
Mint-X Dark Theme

### ISSUE THREAD

**Issue by mfreeman72** [linuxmint/mint-themes#399]
Now that Mint-X is capable of adapting to applications that request a dark mode (which looks *great*, other than my widget problem mentioned in another issue report), could we please get a Mint-X-Dark theme for those of us that want to use dark mode for all applications from time to time, please? Or maybe some kind of switch in Cinnamon that switches on Mint-X's dark mode for all applications?

**Comment by JosephMcc** [comment]
Thanks for the request but currently there are no plans to add a full dark theme in the same way that Mint-Y has. Will keep the idea in mind though.

**Comment by mfreeman72** [comment]
That's too bad. Mint-X is the best Linux GTK theme of any distro! It'd be great to have a dark mode version available, especially since it's basically already there, users just can't access it ourselves.

**Comment by JosephMcc** [comment]
I understand your point but it isn't basically already there. It would require the creation of more things such as full dark variant of the gtk2 and xfce specific themes. That is still a good bit of work that needs to be done and then maintained.

**Comment by mfreeman72** [comment]
Ok. I understand now. I assumed that since it could do dark mode when a program requests it, it was available across the board, for any program to use, regardless of GTK version. Thanks for the info. It still makes me incredibly sad and frustrated, though, that there are no plans to upgrade Mint-X to include dark mode. It's the only Linux theme I've found that I actually love to use, and now seeing it work in dark mode in some apps made it even more beautiful.

**Comment by mfreeman72** [comment]
I found a cool workaround: I copied all the Mint-X theme folders into ~/.themes/, renamed them all to Mint-X-Dark-*, removed the existing gtk.css in each GTK-3.0 and GTK-4.0 folder, and renamed the gtk-dark.css files to gtk.css. This actually accomplishes what I'm looking for in about 90% of my software. The only ones that don't seem to follow the theming are the QT-based programs. I can live with that.
